### PR TITLE
Feature/locations catalog

### PR DESCRIPTION
**Pull request by adamkorynta**

### CODE REVIEW

File: cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestLocationsCatalogController.java [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+import mil.army.usace.hec.cwms.http.client.ApiConnectionInfo;
+import mil.army.usace.hec.cwms.http.client.ServerNotFoundException;
+import mil.army.usace.hec.cwms.radar.client.model.LocationCatalog;
+import mil.army.usace.hec.cwms.radar.client.model.LocationCatalogEntry;
+import org.junit.jupiter.api.Test;
+
+class TestLocationsCatalogController extends TestController {
+
+    @Test
+    void testRetrieveTimeSeriesCatalog() throws IOException {
```
Comment (rma-psmorris) [inline cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestLocationsCatalogController.java:43]: method names are incorrect

File: cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestLocationsCatalogController.java [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+        assertEquals("CA", catalogEntry.getState());
+        assertEquals("Yolo", catalogEntry.getCounty());
+        assertEquals("SPD", catalogEntry.getBoundingOffice());
+        assertEquals("RMA", catalogEntry.getMapLabel());
+        assertTrue(catalogEntry.isActive());
+        assertTrue(catalogEntry.getAliases().isEmpty());
+    }
+
+    @Test
+    void testRetrieveTimeSeriesCatalogPagination() throws IOException {
```
Comment (rma-psmorris) [inline cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestLocationsCatalogController.java:87]: method name

File: cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationAlias.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+import io.swagger.annotations.ApiModelProperty;
+import java.util.Objects;
+
+/**
+ * LocationAlias
+ */
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2021-11-02T12:25:34.578-07:00[America/Los_Angeles]")
+public class LocationAlias {
+    @JsonProperty("name")
+    private final String name = null;
```
Comment (rma-psmorris) [inline cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationAlias.java:20]: group name?

File: cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationAlias.java [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+    @JsonProperty("value")
+    private final String value = null;
```
Comment (rma-psmorris) [inline cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationAlias.java:23]: alias id?

**Review by rma-psmorris — CHANGES_REQUESTED** [review]

File: cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestLocationsCatalogController.java (hunk 1 above)
Comment (adamkorynta) [reply cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestLocationsCatalogController.java]: fixed

File: cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestLocationsCatalogController.java (hunk 2 above)
Comment (adamkorynta) [reply cwms-radar-client/src/test/java/mil/army/usace/hec/cwms/radar/client/controllers/TestLocationsCatalogController.java]: fixed

File: cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationAlias.java (hunk 3 above)
Comment (adamkorynta) [reply cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationAlias.java]: renamed

File: cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationAlias.java (hunk 4 above)
Comment (adamkorynta) [reply cwms-radar-model/src/main/java/mil/army/usace/hec/cwms/radar/client/model/LocationAlias.java]: renamed

**Review by rma-psmorris — APPROVED** [review]
Updates look good to me

**Pull request merged**